src/index.spec.js

```diff
@@ -131,7 +131,7 @@ describe('Mongo', () => {
 
   describe('findUserById', () => {
     it('should return null for not found user', async () => {
-      const ret = await mongo.findUserById('unknowuser');
+      const ret = await mongo.findUserById('589871d1c9393d445745a57c');
```

Comment:
Rather than updating the test wouldn't be better to update your `toMongoID` function to check than the string is a valid Object id like [here](https://github.com/chriso/validator.js/blob/master/src/lib/isMongoId.js#L7)?